src/app/Instance/InstanceDetails/InstanceDetails.tsx

```diff
@@ -0,0 +1,103 @@
+import { format } from "date-fns";
+
+interface InstanceDetailsProps extends Instance {
+  onClosingDetails: (event: React.MouseEvent<HTMLDivElement>) => void;
+}
+
+export const InstanceDetails: React.FunctionComponent<InstanceDetailsProps> = ({
+  id,
+  endpoint,
+  name,
+  owner,
+  onClosingDetails,
+  published_at,
+  submitted_at,
+}) => {
+  const { t } = useTranslation(["openbridgeTempDictionary"]);
+
+  const formatDate = (dateStr: string) => format(new Date(dateStr), "PPPP p");
+
+  return (
+    <DrawerPanelContent widths={{ default: "width_33" }}>
+      <DrawerHead>
+        <Stack>
+          <StackItem>
+            <TextContent>
+              <Text component="small">{t("common.name")}</Text>
+            </TextContent>
+          </StackItem>
+          <StackItem>
+            <TextContent>
+              <Text ouiaId="instance-details-name" component="h2">
+                {name}
+              </Text>
+            </TextContent>
+          </StackItem>
+        </Stack>
+        <DrawerActions>
+          <DrawerCloseButton onClick={onClosingDetails} />
+        </DrawerActions>
+      </DrawerHead>
+      <DrawerContentBody>
+        <DescriptionList isHorizontal isCompact>
+          <DescriptionListGroup>
+            <DescriptionListTerm>{t("common.id")}</DescriptionListTerm>
+            <DescriptionListDescription data-ouia-component-id="instance-details-id">
+              {id}
+            </DescriptionListDescription>
+          </DescriptionListGroup>
+          <DescriptionListGroup>
+            <DescriptionListTerm>{t("common.owner")}</DescriptionListTerm>
+            <DescriptionListDescription data-ouia-component-id="instance-details-owner">
+              {owner}
+            </DescriptionListDescription>
+          </DescriptionListGroup>
+          <DescriptionListGroup>
+            <DescriptionListTerm>{t("common.url")}</DescriptionListTerm>
+            <DescriptionListDescription>
+              <ClipboardCopy
+                data-ouia-component-id="instance-details-endpoint"
+                isReadOnly
+                hoverTip={t("common.copy")}
+                clickTip={t("common.copied")}
+              >
+                {endpoint}
+              </ClipboardCopy>
+            </DescriptionListDescription>
+          </DescriptionListGroup>
+          <DescriptionListGroup>
+            <DescriptionListTerm>{t("common.submittedAt")}</DescriptionListTerm>
+            <DescriptionListDescription data-ouia-component-id="instance-details-submitted-date">
+              {formatDate(submitted_at)}
+            </DescriptionListDescription>
+          </DescriptionListGroup>
+          <DescriptionListGroup>
+            <DescriptionListTerm>{t("common.updatedAt")}</DescriptionListTerm>
+            <DescriptionListDescription data-ouia-component-id="instance-details-published-date">
+              {formatDate(published_at)}
+            </DescriptionListDescription>
```

Comment:
As you may have realized we don't have an "update date" for now. I would use a label reflecting our "Published at" value.

Unfortunately the drawer mockup is not up to date with the discussion we had inside comments.